content/static/js/custom.cfr.js

```diff
@@ -17,4 +20,64 @@ function checkNavbar() {
   } else {
     showNavbar();
   }
-};
\ No newline at end of file
+};
+
+var dummy, dummy2, dummyJSON, txt, x, xmlhttp;
+
+function submitCFR() {
+  dummy = { name1: document.getElementById("cfrName").value, num1: document.getElementById("cfrNum1").value,
+  num2: document.getElementById("cfrNum2").value, num3: document.getElementById("cfrNum3").value};
+  dummyJSON = JSON.stringify(dummy);
+  localStorage.setItem("dummyJSONtest", dummyJSON);
```

Comment:
[NON-ESSENTIAL] We probably shouldn't be saving the data in local storage.

Reply:
This is needed so that when we print the "added <name> <num1> <num2> <num3> to CFR" confirmation the data actually prints. Without it we only print "added  to CFR"